src/rules/converters/space-before-function-paren.ts

```diff
@@ -1,29 +1,32 @@
 import { RuleConverter } from "../converter";
 
-const NOT_SUPPORTED_OPTIONS = ["constructor", "method"];
-
-function isExistedESLintOption(rule: string) {
-    return !NOT_SUPPORTED_OPTIONS.includes(rule);
-}
+const SUPPORTED_OPTIONS: string[] = ["anonymous", "asyncArrow", "named"];
 
 export const convertSpaceBeforeFunctionParen: RuleConverter = tslintRule => {
-    const ruleArguments = tslintRule.ruleArguments.filter(isExistedESLintOption);
-    const notices = NOT_SUPPORTED_OPTIONS.reduce<string[]>((acc, option) => {
-        if (tslintRule.ruleArguments.includes(option)) {
-            acc.push(`Option "${option}" is not supported by ESLint.`);
-        }
-        return acc;
-    }, []);
-
-    return {
-        rules: [
-            {
-                ...(tslintRule.ruleArguments.length !== 0 && {
-                    ruleArguments,
+    const argument: "always" | "never" | { [key: string]: "always" | "never" } | undefined =
+        tslintRule.ruleArguments[0];
+    const ruleName = "space-before-function-paren";
+    if (argument === "always" || argument === "never") {
+        return { rules: [{ ruleArguments: [argument], ruleName }] };
+    } else if (typeof argument === "object" && argument !== null) {
+        const notices = Object.keys(argument)
+            .filter(option => !SUPPORTED_OPTIONS.includes(option))
+            .map(option => `Option "${option}" is not supported by ESLint.`);
+        const filtered = Object.keys(argument)
+            .filter(option => SUPPORTED_OPTIONS.includes(option))
+            .reduce((obj, option) => {
+                return { ...obj, [option]: argument[option] };
+            }, {} as { [key: string]: "always" | "never" });
```

Comment:
Fun fact! This is a little cleaner:

```ts
            .reduce<Record<string, "always" | "never">((obj, option) => {
                    ruleArguments,	                return { ...obj, [option]: argument[option] };
            }, {});
```ts

Reply:
OK. I will stop using `as`.